platforms/documentation/docs/src/docs/userguide/dep-man/getting_started_dep_man.adoc

```diff
@@ -0,0 +1,87 @@
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[dependency_management_in_gradle]]
+= Dependency Management
+
+Software projects typically depend on other libraries to function.
+These libraries can either be sourced from other projects in the same build or from external repositories.
+
+Gradle's dependency management infrastructure provides APIs to declare, resolve, and expose binaries required by and provided by a project.
+
+Understanding dependency management in Gradle is important for structuring projects into components.
+It is also important when you want to reuse existing libraries, or you need to upgrade those libraries while managing their versions.
+
+Let's look at a Java project where the code relies on link:https://mvnrepository.com/artifact/com.google.guava/guava[Guava] which is a suite of Google Core libraries for Java.
+The build file of the project includes the following:
+
+====
+include::sample[dir="snippets/dependencyManagement/introductionDependencies/kotlin",files="build.gradle.kts[tags=dependency-intro]"]
+include::sample[dir="snippets/dependencyManagement/introductionDependencies/groovy",files="build.gradle[tags=dependency-intro]"]
+====
+
+Within the `dependencies` block, there are three things to notice when a dependency is declared:
+
+1. The configuration: `implementation` also known as the scope the dependency is applied to
+2. The coordinates: `com.google.guava:guava` made up of a `group` and an artifact `name` which are uniquely identifiable
+3. The version: `32.1.2-jre` which is only required for published dependencies
+
+Dependencies can be local or external.
+To let Gradle know where to find the dependencies declared, you can use the `repositories{}` block in the <<build_file_basics.adoc#build_file_basics,Build File>>.
+
+Let's expand our example:
+
+====
+include::sample[dir="snippets/dependencyManagement/introductionDependencies/kotlin",files="build.gradle.kts[tags=dependency-repo]"]
+include::sample[dir="snippets/dependencyManagement/introductionDependencies/groovy",files="build.gradle[tags=dependency-repo]"]
+====
+
+In this example, Gradle fetches the `guava` and `juneau-marshall` _dependencies_ from the link:https://repo1.maven.org/maven2/[Maven Central] and link:https://maven.google.com/[Google] _repositories_.
+
+== 1. Declaring Dependencies
+
+You can add external libraries to your Java project such as `com.google.common.base`, a Guava package.
+These libraries are <<declaring_dependencies.adoc#declaring-basic-dependencies, _dependencies_>> of your project.
+They are added using the `dependencies{}` block in your build file.
+
+== 2. Dependency Configurations
+
+You can <<declaring_dependencies.adoc#declaring-basic-dependencies,declare dependencies>> for different scopes. For example, you can declare dependencies that are only used at compile time.
```

Comment:
I find it confusing that the first link in the `2.` section brings you to the `1.` section page.

Reply:
Removed.